che-incubator/chectl
# feat: Deploy operator with che-namespace-editor clusterrole and and binding

### What does this PR do?
Deploys che-operator with che-namespace-editor ClusterRole and ClusterRoleBinding for che-operator ServiceAccount.
It depends on che-operator PR https://github.com/eclipse/che-operator/pull/520

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18272
fixes: https://github.com/eclipse/che/issues/18398


### How to test this PR?

deploy with che-operator from this PR https://github.com/eclipse/che/issues/18272 (or once it's merger into master) and check that che-operator ServiceAccount has cluster-wide permissions to update and get namespaces

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
